contract/contract.go

```diff
@@ -13,61 +14,30 @@ import (
 	"github.com/aergoio/aergo/v2/state"
 	"github.com/aergoio/aergo/v2/types"
 	"github.com/aergoio/aergo/v2/types/dbkey"
-	"github.com/minio/sha256-simd"
 )
 
-/* The preloadWorker optimizes the execution time by preloading the next transaction while executing the current transaction */
-type preloadRequest struct {
-	preloadService int
-	bs             *state.BlockState
-	bi             *types.BlockHeaderInfo
-	next           *types.Tx // the tx to preload the executor for, the next to be executed
-	current        *types.Tx // the tx currently being executed
-}
-
-type preloadReply struct {
-	tx  *types.Tx
-	ex  *executor
-	err error
-}
-
-type preloader struct {
-	requestedTx *types.Tx // the next preload tx to be executed
-	replyCh     chan *preloadReply
-}
-
 var (
-	loadReqCh     chan *preloadRequest
-	preloaders    [2]preloader
 	PubNet        bool
 	TraceBlockNo  uint64
 	bpTimeout     <-chan struct{}
 	maxSQLDBSize  uint64
 	addressRegexp *regexp.Regexp
 )
 
+// These constants indicate the situation in which the contract is executed. BlockFactory refers to execution by a block producer to create a block, from which BlockFactory class calls, and ChainService refers to execution to verify and reflect blocks created by other producers. Depending on the mode, the operations or policies may be different, and currently timeout policy is different.
+// TODO These values are also used to select slots for internal parallel processing. They have multiple roles or meanings so that mit make harder to understand the source code.
 const (
 	BlockFactory = iota
 	ChainService
-	MaxVmService
+	MaxVmService // MaxVmService
```

Comment:
there is no need to repeat the constant name as a comment

Reply:
it seems MaxVmService no more need.

Reply:
This symbol will be removed when the entire logic is refactored.